src/components/EnhancedTable/EnhancedTable.tsx

```diff
@@ -93,7 +93,8 @@ export function EnhancedTable<TRow>({
               height={propHeight || height}
               maxHeight={maxHeight}
               includeHeaders={true}
-              columns={mappedColumns}
+              columns={columns}
+              onHeaderClick={onHeaderClick}
```

Comment:
Wont this cause all columns to be sortable?

Reply:
Nope. I didn't realize that the implementation allowed for you to do `onHeaderClick: false` in a column def

Reply:
I just realized that the type that I just fixed for onHeaderClick doesn't account for booleans, ffs